<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Potaléger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/images.qrc">
    <normaloff>:/images/potaleger.svg</normaloff>:/images/potaleger.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="Info">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>1</y>
          <width>721</width>
          <height>411</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="lDBlabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="palette">
              <palette>
               <active>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>154</red>
                   <green>153</green>
                   <blue>150</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>154</red>
                   <green>153</green>
                   <blue>150</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled/>
              </palette>
             </property>
             <property name="text">
              <string>Base de données :</string>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lDB">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSplitter" name="vSplitter">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <widget class="QSplitter" name="hSplitter">
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="handleWidth">
             <number>4</number>
            </property>
            <widget class="ZoomGraphicsView" name="graphicsView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
             </property>
             <property name="renderHints">
              <set>QPainter::RenderHint::Antialiasing|QPainter::RenderHint::SmoothPixmapTransform|QPainter::RenderHint::TextAntialiasing</set>
             </property>
            </widget>
            <widget class="QFrame" name="frame_2">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>11</y>
                <width>71</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>Thème</string>
              </property>
             </widget>
             <widget class="QComboBox" name="cbTheme">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>10</y>
                <width>111</width>
                <height>23</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>41</y>
                <width>71</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>Taille</string>
              </property>
             </widget>
             <widget class="QComboBox" name="cbFont">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>40</y>
                <width>111</width>
                <height>23</height>
               </rect>
              </property>
             </widget>
            </widget>
           </widget>
           <widget class="QTextEdit" name="pteNotes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="wData">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="toolButton_2">
             <property name="text">
              <string>...</string>
             </property>
             <property name="shortcut">
              <string>Ctrl+Enter, Ctrl+Return</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::RightToLeft</enum>
             </property>
             <property name="text">
              <string>test</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lDBErr">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>18</height>
         </size>
        </property>
        <property name="value">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lVer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="text">
         <string>lVer</string>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lVerBDDAttendue">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="text">
         <string>lVerBDDAttendue</string>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="mFichiers">
    <property name="title">
     <string>Fichiers</string>
    </property>
    <widget class="QMenu" name="mSchema">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Schéma de la BDD</string>
     </property>
     <addaction name="mFKErrors"/>
     <addaction name="mSQLiteSchema"/>
     <addaction name="mFdaTSchema"/>
     <addaction name="mFdaFSchema"/>
     <addaction name="mLaunchers"/>
     <addaction name="mScripts"/>
    </widget>
    <addaction name="mSelecDB"/>
    <addaction name="mCopyDB"/>
    <addaction name="mUpdateSchema"/>
    <addaction name="mCreateDB"/>
    <addaction name="mCreateEmptyDB"/>
    <addaction name="mSchema"/>
    <addaction name="mCloseDB"/>
    <addaction name="mAbout"/>
    <addaction name="mWhatSNew"/>
   </widget>
   <widget class="QMenu" name="mEdition">
    <property name="title">
     <string>Édition</string>
    </property>
    <addaction name="mCloseTab"/>
    <addaction name="mCloseTabs"/>
    <addaction name="mParam"/>
    <addaction name="mNotes"/>
    <addaction name="mRequeteSQL"/>
   </widget>
   <addaction name="mFichiers"/>
   <addaction name="mEdition"/>
  </widget>
  <action name="mSelecDB">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
   </property>
   <property name="text">
    <string>Sélectionner une base de données</string>
   </property>
  </action>
  <action name="mParam">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="emblem-system"/>
   </property>
   <property name="text">
    <string>Paramètres</string>
   </property>
  </action>
  <action name="mCloseTab">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailReplySender"/>
   </property>
   <property name="text">
    <string>Fermer l'onglet courant</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="mCloseTabs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailReplyAll"/>
   </property>
   <property name="text">
    <string>Fermer tous les onglets</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="mCreateDB">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>Créer une BDD avec les données de base</string>
   </property>
  </action>
  <action name="mCreateEmptyDB">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>Créer une BDD vide</string>
   </property>
  </action>
  <action name="mCopyDB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy"/>
   </property>
   <property name="text">
    <string>Faire une copie de la BDD</string>
   </property>
  </action>
  <action name="mAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>A propos...</string>
   </property>
  </action>
  <action name="mUpdateSchema">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="system-reboot"/>
   </property>
   <property name="text">
    <string>Mettre à jour le schéma de BDD</string>
   </property>
  </action>
  <action name="mNotes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
  </action>
  <action name="mWhatSNew">
   <property name="icon">
    <iconset theme="dialog-question"/>
   </property>
   <property name="text">
    <string>Quoi d'neuf ?</string>
   </property>
  </action>
  <action name="mRequeteSQL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Requête SQL</string>
   </property>
  </action>
  <action name="mFKErrors">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
   <property name="text">
    <string>Erreurs d'intégrité</string>
   </property>
  </action>
  <action name="mSQLiteSchema">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentPageSetup"/>
   </property>
   <property name="text">
    <string>Schéma SQLite</string>
   </property>
  </action>
  <action name="mFdaTSchema">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentPageSetup"/>
   </property>
   <property name="text">
    <string>Schéma additionnel (tables et vues)</string>
   </property>
  </action>
  <action name="mFdaFSchema">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentPageSetup"/>
   </property>
   <property name="text">
    <string>Schéma additionnel (champs)</string>
   </property>
  </action>
  <action name="mLaunchers">
   <property name="icon">
    <iconset theme="applications-development"/>
   </property>
   <property name="text">
    <string>Lanceurs</string>
   </property>
  </action>
  <action name="mCloseDB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="system-log-out"/>
   </property>
   <property name="text">
    <string>Se déconnecter de la BDD</string>
   </property>
  </action>
  <action name="mScripts">
   <property name="icon">
    <iconset theme="application-x-executable"/>
   </property>
   <property name="text">
    <string>Scripts</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>classes/zoomgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
